vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -56,11 +61,16 @@ type internal Settings [<ImportingConstructor>](store: SettingsStore) =
               AlwaysPlaceOpensAtTopLevel = false
               UnusedOpens = true }
 
+        store.RegisterDefault
+            { EnableInMemoryCrossProjectReferences = true
+              ProjectCheckCacheSize = 3 }
```

Comment:
Default should be 200 since that's what we shipped in VS2017 (i.e. let's not change the default in this PR)